traits/trait_base.py

```diff
@@ -211,20 +211,27 @@ def safe_contains(value, container):
         return False
 
 
-def collection_default(collection):
-    """ Get the first item of a collection, returning None if empty.
+def enum_default(values):
+    """ Get the first item of an EnumType, returning None if empty.
```

Comment:
Replace "EnumType" with "Enum subclass"? (It's not clear to me what `EnumType` refers to here.)

Reply:
Ah, I see; `EnumType` means any of the types listed in `EnumTypes`. Maybe we can just rephrase in plain English?